changelog.d/11867.feature

```diff
@@ -0,0 +1 @@
+Stabilize MSC3231.
```

Comment:
I think this could use a bit more detail:

1. Could you add a markdown link to the MSC?
2. We should explicitly say something like:

    > Client implementations using `m.login.registration_token` should switch to the stable identifiers:
    > * `org.matrix.msc3231.login.registration_token` in query parameters and request/response bodies becomes `m.login.registration_token`
    > * `/_matrix/client/unstable/org.matrix.msc3231/register/org.matrix.msc3231.login.registration_token/validity` becomes `/_matrix/client/v1/register/m.login.registration_token/validity`